srcpkgs/docker/template

```diff
@@ -1,138 +1,10 @@
 # Template file for 'docker'
 pkgname=docker
```

Comment:
This should be a subpackage of either `docker-cli` or `moby`, which has the benefits of reducing maintenance burden and of always pegging the version to that of one of the two major components this pulls in.

The best choice is making this a subpackage of `docker-cli` for two reasons:
1. In keeping with the current unified template, one expects `docker` to effectively provide a turnkey `docker` installation, and the component the user is most likely to associate with `docker` is the CLI tooling. Thus, these versions should remain consisten even if the `moby` version diverges.
2. If `docker-cli` depends on `moby` (see above), putting a `docker` subpackage in the `moby` template will create a build cycle.

Reply:
See my larger response.